src/dds/rs-dds-option.cpp

```diff
@@ -9,13 +9,24 @@
 namespace librealsense {
 
 
+static option_range range_from_realdds( std::shared_ptr< realdds::dds_option > const & dds_opt )
+{
+    if( dds_opt->get_minimum_value().is_number() && dds_opt->get_maximum_value().is_number()
```

Comment:
Shouldn't also check if `get_default_value().is_number()`?

Reply:
Part of the range can be `null` -- I don't enforce a complete range (so you can just give a minimum, for example) right now.
So here it's checking that it has a complete range. A range requires a default value, and the types should be checked.
Still, you may be right. I haven't worked out all the kinks yet, and this specific code may need work for non-range values (strings, booleans, string "choices", etc.).